src/SolutionTemplate/UnoSolutionTemplate/Wasm/Wasm.vstemplate

```diff
@@ -20,7 +20,13 @@
 	<Project TargetFileName="$ext_safeprojectname$.Wasm.csproj" File="UnoQuickStart.Wasm.csproj" ReplaceParameters="true">
 	  <ProjectItem ReplaceParameters="true" TargetFileName="Program.cs">Program.cs</ProjectItem>
 	  <ProjectItem ReplaceParameters="true">LinkerConfig.xml</ProjectItem>
+	  <ProjectItem ReplaceParameters="true" TargetFileName="manifest.json">manifest.json</ProjectItem>
```

Comment:
Let's only add the support for PWA in the net6 default template. The UWP template is less used now, and it does not have a configuration dialogs. Developers can add it manually afterwards.